### PR TITLE
Enable strict-transport-security for kube-apiserver

### PR DESCRIPTION
**Pull request by jmcmeek**
Adds strict-transport-security-directives to the KubeApiServerConfig.

### CODE REVIEW

File: assets/kube-apiserver/config.yaml [hunk 1]
```diff
@@ -199,6 +199,8 @@ apiServerArguments:
   - /etc/kubernetes/secret/server.crt
   tls-private-key-file:
   - /etc/kubernetes/secret/server.key
+  strict-transport-security-directives:
```
Comment (jonesbr17) [inline assets/kube-apiserver/config.yaml:202]: Can you move this up a few lines to maintain the alphabetical sorting of this list? Other than that, this looks good

**Review by jonesbr17 — CHANGES_REQUESTED** [review]

File: assets/kube-apiserver/config.yaml (hunk 1 above)
Comment (jmcmeek) [reply assets/kube-apiserver/config.yaml]: moved.

**Review by jonesbr17 — APPROVED** [review]
/lgtm
/approved

**Pull request merged**